FW-1019 Create script to setup language for local dev work

NolanVH opened the pull request:
These changes add a script to create a language with some basic words, phrases, and an alphabet for frontend development.  This script could be added to in the future as more items are needed.

To run the script after starting up the backend/initial setup, run the command from the frontend directory:
```npm run local:language:setup```
To remove the created language run the command from the frontend directory:
```npm run local:language:teardown```

dy2288 commented:
Can one of the admins verify this patch?

davidlacho commented:
Checked again this morning and stack trace above was most likely due to cruddy wifi. Script works fine, just need to tackle the issue of dir for cloning :)

NolanVH commented:
> Checked again this morning and stack trace above was most likely due to cruddy wifi. Script works fine, just need to tackle the issue of dir for cloning :)

Updated the pull request to include the changes requested. Batch/Utils is now stored in a temp folder and I have added checking to ensure the git clone completes sucessfully.

Thanks @davidlacho!
